lms/lmsdb/models.py

```diff
@@ -708,7 +709,7 @@ def create_comment(
         cls, text: str, flake_key: Optional[str] = None,
     ) -> 'CommentText':
         instance, _ = CommentText.get_or_create(
-            **{CommentText.text.name: text},
+            **{CommentText.text.name: html.escape(text)},
```

Comment:
Can you please add a single test that ensures the escaping?

Reply:
Sure